spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -291,25 +295,25 @@ class SpdxExpressionTest : WordSpec() {
             "not change an expression already in DNF" {
                 val spdxExpression = "(a AND b) OR (c AND d)".toSpdx()
 
-                spdxExpression.disjunctiveNormalForm() shouldBe spdxExpression
+                spdxExpression.disjunctiveNormalForm() shouldBeString spdxExpression.toString()
```

Comment:
Wouldn't it be easier to just call "toString()" on the left-hand side (i.e. on `disjunctiveNormalForm()`) and then keep the string (but without the `.toSpdx()`) on the right-hand side?

Reply:
I went one step further and duplicated the string to align with the style of the other tests.